simply/battery.py

```diff
@@ -0,0 +1,26 @@
+class Battery:
+    """Class which implements all raw functionality around the battery"""
```

Comment:
I think I understand your point, but if we differentiate in this case, is not every thing we handle in every kind of model a "representation", e.g, a market representation, actor representation.

Reply:
Yes, maybe representation does not make it clearer, was just trying to express that in this case it is not only a physical model which represents its behavior, but also rather a representation of a battery model for planning rather than a physical model. But we can leave this comment away here, as it is rather a question of how an object of this class is applied within the actor and have the comment only there.